lib/features/grid-snapping/Grid.js

```diff
@@ -80,6 +93,23 @@ Grid.prototype._init = function() {
   });
 };
 
+Grid.prototype._centerGridAroundViewbox = function(viewbox) {
+  if (!viewbox) {
+    viewbox = this._canvas.viewbox();
+  }
+
+  if (!viewbox) {
+    return;

```

Comment:
Can this ever happen? `Canvas#viewbox` always returns something so I think this check is unnecessary.

Reply:
👍